_includes/components/header-and-footer/SubLinks/index.html

```diff
@@ -0,0 +1,39 @@
+{% load define_action %}
+{% comment %} Determine whether children should show in sidebar {% endcomment %}
+{% if active == 'true' %}
+  {% assign ' is-currentPage' as parentActiveClass %}
+{% else %}
+  {% assign '' as parentActiveClass %}
+{% endif %} 
+
+
+<div class="SubLinks{{ parentActiveClass }}">
+  <ul class="SubLinks-subItems">
+
+    {% for item in items %}
+
+      {% comment %} Determine whether children should show in sidebar {% endcomment %}
+      {% if selected_sidebar and selected_sidebar == item.id %}
+        {% assign ' is-active' as activeClass %}
+      {% else %}
+        {% assign '' as activeClass %}
+      {% endif %}
+
+      {% comment %} Determine whether link active state should be determined by JavaScript {% endcomment %}
+      {% if item.connected %}
+        {% assign ' data-connected-id='|add:item.connected as dataAttribute %}
```

Comment:
I don't understand - where do the quotes come from? Something tells me this is a hack and not a solution that's easy to maintain.

```diff
--- a/_includes/components/header-and-footer/SubLinks/index.html
+++ b/_includes/components/header-and-footer/SubLinks/index.html
@@ -21,7 +21,7 @@
 
       {% comment %} Determine whether link active state should be determined by JavaScript {% endcomment %}
       {% if item.connected %}
-        {% assign ' data-connected-id="'|add:item.connected|add:'"' as dataAttribute %}
+        {% assign ' data-connected-id='|add:item.connected as dataAttribute %}
```

Reply:
I think quotes come from django template render. it auto added the quotes to value of attributes

Reply:
right - I doubt that but ok - let's see if this becomes a problem and go with this for now.